packages/rest/test/integration/http-handler.integration.ts

```diff
@@ -272,6 +264,78 @@ describe('HttpHandler', () => {
         });
     });
 
+    it('rejects unsupported request body', () => {
+      logErrorsExcept(415);
+      return client
+        .post('/show-body')
+        .set('content-type', 'application/xml')
+        .send('<key>value</key>')
+        .expect(415, {
+          error: {
+            message: 'Content-type application/xml is not supported.',
+            name: 'UnsupportedMediaTypeError',
+            statusCode: 415,
+          },
+        });
+    });
+
+    it('rejects over-limit request form body', () => {
```

Comment:
Uh, I did not anticipate how complex this test will become when I asked to add it. Considering that we didn't have this kind of tests before this pull request (in the current `master`), I am ok to leave them out of this PR.

In case you prefer to keep these two new tests, then please:
 - make it more clear that `EPIPE` is one of the expected outcomes of the test
 - add a check to verify that the controller method/route handler was not invoked